fluid/fluid.cpp

```diff
@@ -584,10 +586,17 @@ void Fluid::updatePatchList()
                   patch->bank = p->get_banknum() + bankOffset;
                   patch->prog = p->get_num();
                   patch->name = p->get_name();
-                  patches.append(patch);
+                  patchNums[patch->prog].append(patch);
                   }
             bankOffset += (banks + 1);
             }
+
+      // Order by patch number first instead of by bank first
+      for (QList<MidiPatch*> num : patchNums) {
+         for (MidiPatch* p : num)
+            patches.append(p);
+         }
+
       /* try to set the correct presets */
```

Comment:
indentation issue, should be 6 spaces per level

Reply:
Not in fluid it isn't, they're using a different number.

Reply:
seems the surrounding code is using something different

Reply:
Oh, sorry, my mistake. Will fix.

Reply:
Fixed now.